files/image_config/logrotate.d/rsyslog

```diff
@@ -30,14 +28,14 @@
 {
     rotate 4
     weekly
-    size 100M
+    maxsize 100M
     missingok
     notifempty
     compress
     delaycompress
     sharedscripts
     postrotate
-        invoke-rc.d rsyslog rotate > /dev/null
```

Comment:
>nvoke-rc.d rsyslog rotate > /dev/null [](start = 9, length = 37)

in /etc/init.d/rsyslog, rotate is doing the kill HUP. why not use invoke-rc.d rsyslog rotate?